### ISSUE TITLE
[Feature]: rocblas-bench do not use (x,y) to express complex numbers

### ISSUE THREAD

**Issue by littlewu2508** [ROCmSoftwarePlatform/rocBLAS#1257]
### Is your feature request related to a problem? Please describe.
I use rocblas-bench combined with csvlook to get a visualized result, for example:
```
rocblas-bench -f gemm -r f32_r --transposeA N --transposeB N -m 4096 -n 4096 -k 4096 --alpha 1 --lda 4096 --ldb 4096 --beta 0 --ldc 4096 --verify 1 | tail -2 | csvlook -I
| transA | transB | M    | N    | K    | alpha | lda  | beta | ldb  | ldc  | rocblas-Gflops | us     | CPU-Gflops | CPU-us | norm_error_1 |
| ------ | ------ | ---- | ---- | ---- | ----- | ---- | ---- | ---- | ---- | -------------- | ------ | ---------- | ------ | ------------ |
| N      | N      | 4096 | 4096 | 4096 | 1     | 4096 | 0    | 4096 | 4096 |  11011.9       |  12481 | 913.252    | 150494 | 1.19257e-06  |
```

However when benchmarking complex floating point operations, the results use (x,y) to express complex number:
```
rocblas-bench -f gemm -r f64_c --transposeA N --transposeB N -m 4096 -n 4096 -k 4096 --alpha 1 --lda 4096 --ldb 4096 --beta 0 --ldc 4096 --verify 1 | tail -2
transA,transB,M,N,K,alpha,lda,beta,ldb,ldc,rocblas-Gflops,us,CPU-Gflops,CPU-us,norm_error_1
N,N,4096,4096,4096,(1,0),4096,(0,0),4096,4096, 3275.88, 167820,414.319,1.32689e+06,2.19498e-15
```

which cause csvlook think there are more columns in the second line.:
```
agate/table/from_csv.py:74: RuntimeWarning: Error sniffing CSV dialect: Could not determine delimiter
ValueError: Row 0 has 17 values, but Table only has 15 columns.
```

Could this be changed a bit, like replacing x+yi ?

### Library context
| Software | version |
|-----|-----|
| rocblas | ROCm-5.1.3 |

**Comment by TorreZuk** [comment]
Thanks @littlewu2508 for your suggestion, I agree that convention makes parsers more trouble than it is worth.  The problem is we may already have this handled so need to update our scripts as well, will review and get back to you.

**Comment by TorreZuk** [comment]
The complex output for rocblas-bench will be "(real: maginary)" after develop branch commit 79fcb2b107ceb1e201765c873de80be274bc079b
Output elsewhere has not change to minimize any effects on other parsers.  If this causes no trouble we can consider changing to this format for all complex number output sometime in the future.

**Comment by TorreZuk** [comment]
The change to output complex values without comma from client rocblas-bench is in the available 5.4 release so closing issue.
